Makefile

```diff
@@ -1,284 +1,25 @@
-.PHONY: help docs test test_es test_no_es test_force_db tdd test_failed initialize_db populate_data update_code update_deps update_db update_assets full_init full_update reindex flake8 update_docker initialize_docker shell debug
-NUM_ADDONS=10
-NUM_THEMES=$(NUM_ADDONS)
-
-APP=src/olympia/
-
-# Get the name of the Makefile's directory for the docker container base name.
-mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
-current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
-
-COMPOSE_PROJECT_NAME?=$(shell echo "${current_dir}" | tr -d '-' | tr -d '_')
-DOCKER_NAME="${COMPOSE_PROJECT_NAME}_web_1"
-DOCKER_NAME_WORKER="${COMPOSE_PROJECT_NAME}_worker_1"
-
-UNAME_S := $(shell uname -s)
+.PHONY: help docs test test_es test_no_es test_force_db tdd test_failed initialize_db populate_data update_deps update_db update_assets initialize update reindex flake8 shell debug
 
 IN_DOCKER = $(wildcard /addons-server-centos7-container)
 
-NPM_ARGS :=
-
-ifneq ($(NPM_CONFIG_PREFIX),)
-	NPM_ARGS := --prefix $(NPM_CONFIG_PREFIX)
+ifneq ($(IN_DOCKER),)
+	SUB_MAKEFILE = Makefile-os
 endif
-
-NODE_MODULES := $(NPM_CONFIG_PREFIX)node_modules/
-STATIC_JS := static/js/node_lib/
-STATIC_JQUERY_UI := static/js/node_lib/ui/
-
-NODE_LIBS_JS := \
-less/dist/less.js \
-jquery/dist/jquery.js
-
-NODE_LIBS_JQUERY_UI := \
-jquery-ui/ui/version.js \
-jquery-ui/ui/data.js \
-jquery-ui/ui/disable-selection.js \
-jquery-ui/ui/ie.js \
-jquery-ui/ui/keycode.js \
-jquery-ui/ui/escape-selector.js \
-jquery-ui/ui/labels.js \
-jquery-ui/ui/jquery-1-7.js \
-jquery-ui/ui/plugin.js \
-jquery-ui/ui/safe-active-element.js \
-jquery-ui/ui/safe-blur.js \
-jquery-ui/ui/scroll-parent.js \
-jquery-ui/ui/focusable.js \
-jquery-ui/ui/tabbable.js \
-jquery-ui/ui/unique-id.js \
-jquery-ui/ui/position.js \
-jquery-ui/ui/widget.js \
-jquery-ui/ui/widgets/menu.js \
-jquery-ui/ui/widgets/mouse.js \
-jquery-ui/ui/widgets/autocomplete.js \
-jquery-ui/ui/widgets/datepicker.js \
-jquery-ui/ui/widgets/sortable.js
+	SUB_MAKEFILE = Makefile-docker
+include $(SUB_MAKEFILE)
```

Comment:
So it could be that this was broken since forever but at my machine it doesn't include the correct makefile, mind trying 

```bash
IN_DOCKER = $(wildcard /addons-server-centos7-container)

ifneq ("$(IN_DOCKER)","")
	SUB_MAKEFILE = Makefile-docker
else
	SUB_MAKEFILE = Makefile-os
endif

include $(SUB_MAKEFILE)
```

which now includes the -os on my host and -docker inside docker.

```bash
[root@b895fc006666 code]# make help
Please use 'make <target>' where <target> is one of the following commands.
Commands that are designed to be run in either the container or the host:
  docs              to builds the documentation
  flake8            to run the flake8 linter
  update_code       to update the git repository
Commands that are designed be run in the container:
  initialize        to init the dependencies, the database, and assets
  update            to update the dependencies, the database, and assets
  initialize_db     to create a new database
  populate_data     to populate a new database
  reindex           to reindex everything in elasticsearch, for AMO
  update_deps       to update the pythondependencies
  update_db         to run the database migrations
Check the Makefile to know exactly what each target is doing.
```

and on the host

```bash
[git:eviljeff/split-makefile+ (olympia)] ~/Projects/olympia> make help
Please use 'make <target>' where <target> is one of the following commands.
Commands that are designed to be run in either the container or the host:
  docs              to builds the documentation
  flake8            to run the flake8 linter
  update_code       to update the git repository
Commands that are designed to be run in the host:
  debug             to connect to a running addons-server docker for debugging
  djshell           to connect to a running addons-server docker django shell
  dbshell           to connect to a running addons-server docker database shell
  make              to connect to a running addons-server docker and run make ARGS
  shell             to connect to a running addons-server docker shell
  tdd               to run the entire test suite, but stop on the first error
  test              to run the entire test suite
  test_es           to run the ES tests
  test_failed       to rerun the failed tests from the previous run
  test_force_db     to run the entire test suite with a new database
  test_no_es        to run all but the ES tests
  initialize_docker to initialize a docker image
  update_docker     to update a docker image
Check the Makefile to know exactly what each target is doing.
```